src/posix.h

```diff
@@ -130,6 +131,7 @@ extern ssize_t p_pwrite(int fd, const void *data, size_t size, off64_t offset);
 
 #define p_close(fd) close(fd)
 #define p_umask(m) umask(m)
+#define p_mktemp(p) mktemp(p)
```

Comment:
Windows have different implementations for `p_mkstemp` depending on `_MSC_VER`, is that needed for `p_mktemp` too?